src/pretix/plugins/socialauth/urls.py

```diff
@@ -3,5 +3,7 @@
 from . import views
 
 urlpatterns = [
-    url(r'^oauth_return$', views.oauth_return, name='social.oauth.return')
+    url(r'^oauth_login/(?P<provider>[a-zA-Z]+)/$', views.oauth_login, name='social.oauth.login'),
```

Comment:
There is no reason to use old method for new code.
Follow this new style instead: https://docs.djangoproject.com/en/5.1/topics/http/urls/